ldaptive/src/main/java/org/ccci/idm/user/ldaptive/dao/LdaptiveUserDao.java

```diff
@@ -170,6 +171,7 @@ private int enqueueAllByFilter(@Nonnull final Collection<User> users, @Nullable
             conn.open();
             SearchOperation search = new SearchOperation(conn);
             final SearchRequest request = new SearchRequest(this.baseSearchDn, filter);
+            request.setReturnAttributes("*", LDAP_ATTR_PASSWORDCHANGEDTIME);
```

Comment:
I had to dig a little to find the setting and the values I needed.

here's the relevant values for reference: https://github.com/vt-middleware/ldaptive/blob/master/core/src/main/java/org/ldaptive/ReturnAttributes.java#L17